.travis.yml

```diff
@@ -13,17 +13,40 @@ branches:
 env:
     global:
         - RUST_BACKTRACE=1
-    matrix:
-        - RUST_TOOLCHAIN=stable TARGET=rustfmt
-        - RUST_TOOLCHAIN=nightly TARGET=wasm
-        - RUST_TOOLCHAIN=nightly TARGET=native
 
 before_install:
     # Check how much space we've got on this machine.
     - df -h
 
-script:
-    - .maintain/ci/script.sh
+jobs:
+  include:
```

Comment:
It takes quite long time, does it do env clean before each stage? I mean, does every stage require rebuilding?

Reply:
I cache `target` to solve this issue.